its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -581,24 +566,6 @@ void testRazorCompilationNet6WithSourceGenerators() throws IOException {
     validateRazorProject(projectName);
   }
 
-  @Test
-  void testRazorCompilationNet7WithSourceGenerators() throws IOException {
-    // This is not supported for versions older than VS 2022
-    assumeTrue(TestUtils.getMsBuildPath(ORCHESTRATOR).toString().contains("2022"));
-    String projectName = "RazorWebApplication.net7.withSourceGenerators";
-    assertProjectFileContains(projectName, "<UseRazorSourceGenerator>true</UseRazorSourceGenerator>");
-    validateRazorProject(projectName);
-  }
-
-  @Test
-  void testRazorCompilationNet7WithoutSourceGenerators() throws IOException {
-    // This is not supported for versions older than VS 2022
-    assumeTrue(TestUtils.getMsBuildPath(ORCHESTRATOR).toString().contains("2022"));
-    String projectName = "RazorWebApplication.net7.withoutSourceGenerators";
-    assertProjectFileContains(projectName, "<UseRazorSourceGenerator>false</UseRazorSourceGenerator>");
```

Comment:
Do we have an equivalent test still around? This looks like it also tests the target files that come with the scanner.